.github/workflows/build-and-release.yml

```diff
@@ -44,10 +43,21 @@ jobs:
         run: |
           python setup.py sdist bdist_wheel
 
+  push_image_to_dockerhub:
+    runs-on: ubuntu-latest
+    if: ${{ success() && github.event_name == 'push' }}
+    steps:
+      - name: Login to DockerHub
+        uses: docker/login-action@v1
+        with:
+          registry: registry.cern.ch
+          username: ${{ secrets.HARBOR_USERNAME }}
+          password: ${{ secrets.HARBOR_PASSWORD }}
 
-      - name: Publish package
-        if: ${{ success() && github.event_name == 'push'  }}
-        uses: pypa/gh-action-pypi-publish@v1.3.1
+      - name: Build and push
+        id: docker_build
+        uses: docker/build-push-action@v2
         with:
-          user: __token__
-          password: ${{ secrets.pypi_password }}
+          context: ./docker/opensearch/
+          push: true
+          tags: registry.cern.ch/cern-sis/inspirehep/refextract:latest,registry.cern.ch/cern-sis/inspirehep/refextract:${{ github.sha }}
```

Comment:
maybe add a step to update the `kubernetes` repo as in https://github.com/inspirehep/inspirehep/blob/29513546b20e39e07e3624ec3e58ab647e896cb9/.github/workflows/release.yml#L80-L86 ?

Reply:
maybe we can do it at some point but we have some other apps that are using custom images and we didn't want to implement that feature for them afair (eg. notification operator)